package.json

```
@@ -38,6 +38,7 @@
     "@wordpress/data-controls": "^2.2.3",
     "@wordpress/date": "3.9.0",
     "@wordpress/dom-ready": "^3.2.1",
+		"@wordpress/eslint-plugin": "^17.13.0",
```

Nit: whitespace issue.
```suggestion
    "@wordpress/eslint-plugin": "^17.13.0",
```